lua/gruvbox_crisp/groups.lua

```
@@ -0,0 +1,271 @@
+    PreCondit    = { fg = p.kw_ctrl },
+    Type         = { fg = p.type },
+    StorageClass = { fg = p.kw_ctrl },
+    Structure    = { fg = p.kw_ctrl },
+    Typedef      = { fg = p.kw_ctrl },
+    Special      = { fg = p.link },
+    SpecialChar  = { fg = p.link },
+    Tag          = { fg = p.operator },
+    Delimiter    = { fg = p.punct },
+    SpecialComment = { fg = p.comment, italic = bool(o.italics.comments) },
+    Debug        = { fg = p.red },
+    Underlined   = { underline = true },
```

Luacheck (1.2.0)

[warning] value assigned to field 'Underlined' is overwritten on line 255 before use

(W314)